src/pages/creer-un-jeu/mimique/3.tsx

```diff
@@ -13,6 +13,15 @@ const MimiqueStep3 = () => {
   const router = useRouter();
   const { activity, updateActivity } = React.useContext(ActivityContext);
 
+  const data = (activity?.data as MimicsData) || null;
+
+  const isValid = React.useMemo(() => {
+    if (activity !== null && activity.content.filter((c) => c.value.length > 0 && c.value !== '<p></p>\n').length === 0) {
+      return false;
+    }
+    return true;
+  }, [activity]);
```

Comment:
Same here, the goal would be to know whether step 1 is valid and whether step 2 is valid. Considering you have a function to check if a mimic is valid or not:
```tsx
const errorSteps = React.useMemo(() => {
    const errors: number[] = [];
    if (!isMimicValid(data.game1)) {
         errors.push(0); // step of mimic 1
    }
    if (!isMimicValid(data.game2)) {
         errors.push(1); // step of mimic 2
    }
    return errors;
  }, [data]);
```

Reply:
I made the change here too.